android/app/src/main/res/layout/item_category_description.xml

```diff
@@ -18,6 +18,7 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:textAppearance="@style/MwmTextAppearance.Body3"
+      android:visibility="gone"
```

Comment:
Looks like the `author` text view is never uses, so it's probably better to remove the dead code here and in BookmarkListAdapter.java

Reply:
Removed This From Codebase